seed.js

```diff
@@ -0,0 +1,142 @@
+const { green, red } = require('chalk');
+const { db } = require('./server/db');
+
+const products = [
+  {
+    name: 'Panda Mug',
+    description: 'cup',
+    imageUrl: '/images/mug.jpeg',
+    stock: 15,
+    price: 5,
+    category: [
+        { name: 'kitchen'},
+      ]
+    },
+  {
+    name: 'Panda at the back',
+    description: 'backpack',
+    imageUrl: '/images/mug.jpeg',
+    stock: 10,
+    price: 35,
+    category: [
+        { name: 'accesories'},
+        { name: 'home'},
+      ]
+  },
+  {
+    name: 'Panda family',
+    description: 'toys',
+    imageUrl: '/images/toys.jpg',
+    stock: 5,
+    price: 15,
+    category: [
+        { name: 'toys'},
+      ]
+  },
+  {
+    name: 'Cute Keychain',
+    description: 'keychain',
+    imageUrl: '/images/keychain.jpg',
+    stock: 4,
+    price: 9,
+    category: [
+        { name: 'accesories'},
+      ]
+  },
+  {
+    name: 'Panda on my phone',
+    description: 'phonecase',
+    imageUrl: '/images/phonecase.jpg',
+    stock: 8,
+    price: 16,
+    category: [
+        { name: 'accesories'},
+      ]
+  },
+  {
+    name: 'Panda umbrella',
+    description: 'umbrella',
+    imageUrl: '/images/umbrella.jpg',
+    stock: 1,
+    price: 14,
+    category: [
+        { name: 'home'},
+        { name: 'accesories'},
+      ]
+  },
+  {
+    name: 'Comfy panda',
+    description: 'pillow',
+    imageUrl: '/images/pillow.jpg',
+    stock: 6,
+    price: 20,
+    category: [
```

Comment:
since category on products is a foreign key, instead of creating a new object at a new memory address, it might be something like 
```suggestion
    category: categories[1]
```
to indicate the 'home' object of categories array. this goes for all the others as well